test/library/packages/ProtobufProtocolSupport/anyRunner.skipif

```diff
@@ -4,7 +4,7 @@ from distutils.spawn import find_executable
 
 import os
 
-chplPath = os.environ['CHPL_HOME'] + "/bin/" + os.environ['CHPL_BIN_SUBDIR']
+chplPath = os.environ['CHPL_HOME'] + "/bin/" + os.environ['CHPL_HOST_BIN_SUBDIR']
```

Comment:
Nit, but if we're making changes would be nicer to use `os.path.join`

Reply:
lol, I GUESS :P

Reply:
I'm so glad all the other ones symlinks